Resources/config/dbal.xml

```diff
@@ -43,7 +43,9 @@
             <argument>%doctrine.dbal.connection_factory.types%</argument>
         </service>
 
-        <service id="doctrine.dbal.connection" class="Doctrine\DBAL\Connection" factory-service="doctrine.dbal.connection_factory" factory-method="createConnection" abstract="true" />
+        <service id="doctrine.dbal.connection" class="Doctrine\DBAL\Connection" abstract="true">
```

Comment:
This change requires locking to `symfony/dependency-injection` at https://github.com/doctrine/DoctrineBundle/blob/c12410ca351c7742ced38fe9441f6b84be467e97/composer.json#L26